pkg/products/rhsso/reconciler.go

```diff
@@ -494,15 +702,15 @@ func getUserDiff(keycloakUsers []*aerogearv1.KeycloakUser, openshiftUsers []user
 
 	var deleted []int
 	for i, kcUser := range keycloakUsers {
-		if kcUser.UserName != CustomerAdminUser.UserName && !OsUserInKc(openshiftUsers, kcUser) {
```

Comment:
is this a typo? `OsUserInKc` => `IsUserInKc`?

Reply:
I think this stands for `Openshift user in Keycloak`